tutorials/exporting-stackdriver-elasticcloud/index.md

```diff
@@ -0,0 +1,337 @@
+
+## Costs
+
+This tutorial uses billable components of Google Cloud Platform (GCP), including Google Compute Engine.
+
+New GCP users might be eligible for a [free trial](https://cloud.google.com/free-trial).
+
+## Configure GCP resources
+
+The high-level steps in this section are:
+
+1. Create a user-managed service account
+1. Create a VM for Logstash
+1. Create a Cloud Pub/Sub topic
+1. Create a Stackdriver log sink and subscribe it to the Cloud Pub/Sub topic
+
+## Enable APIs
+
+Log in or sign up for [Google Cloud Platform](https://cloud.google.com), then open the [Cloud Console](https://console.cloud.google.com).
+
+The examples in this document use the `gcloud` command-line inteface. Cloud APIs must be enabled via the
+[Services and APIs page](https://console.cloud.google.com/apis/dashboard) in the console before they can be used
+with `gcloud`. To perform the steps in this tutorial, enable the following APIs: 
+
+* Compute Engine
+* Cloud Pub/Sub
+* Identity and Access Management (IAM)
+* Google Stackdriver
+
+![Enable Cloud APIs](https://storage.googleapis.com/gcp-community/tutorials/exporting-stackdriver-elasticcloud/enable_apis.png)
+
+## Activate Google Cloud Shell
+
+The GCP Console provides an interactive shell that includes the `gcloud` command-line interface. At the top right corner of
+the page, click the **Activate Google Cloud Shell** button.
+
+![alt text](https://storage.googleapis.com/gcp-community/tutorials/exporting-stackdriver-elasticcloud/cloud_shell_icon.png)
+
+## Create a service account
+
+GCP [best practices](https://cloud.google.com/vpc/docs/firewalls#service-accounts-vs-tags) suggest using a 
+service account to configure security controls to a VM. A service account is useful for a VM to determine which other GCP 
+resources can be accessed by the VM and its applications, and which firewall rules should be applied to the VM.
+
+While credentials can be created to be used by a service account, this step is not necessary when the service account is
+attached to a VM running on Google Compute Engine. Google manages the keys, and applications can
+[retrieve the credentials securely](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#authenticating_applications_using_service_account_credentials)
+with the metadata service.
+
+1.  Create a service account to attach to the VM:
+
+        gcloud iam service-accounts create logstash \
+            --display-name="Logstash to Stackdriver"
+
+    **Expected response:**
+
+        Created service account [logstash].
+    
+1.  Provide permissions to access Cloud Pub/Sub using the `pubsub.editor`
+    [predefined IAM role](https://cloud.google.com/pubsub/docs/access-control#tbl_roles). _Editor_ permissions are required
+    for version 1.1.0 of Logstash Pub/Sub plugin. This appears to have been
```

Comment:
flagging this mention of version 1.1.0 as part of conversation in a later comment